### PR TITLE
Nightly-job

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

File: pmm-app/tests/verifyAWSRDSMySQLInstance_test.js [hunk 1]
```diff
@@ -57,8 +57,10 @@ Scenario(
     I.amOnPage(dashboardPage.mySQLInstanceOverview.urlWithRDSFilter);
     dashboardPage.waitForDashboardOpened();
     await dashboardPage.expandEachDashboardRow();
-    await dashboardPage.verifyThereAreNoGraphsWithNA();
-    await dashboardPage.verifyThereAreNoGraphsWithoutData(3);
+    // Sevices has N/A - increasing to 1
+    await dashboardPage.verifyThereAreNoGraphsWithNA(1);
+    // So many graphs have No Data - skipping the check
```
Comment (puneet0191) [inline pmm-app/tests/verifyAWSRDSMySQLInstance_test.js:62]: please create a ticket to investigate, and add that as To-Do, so many graphs is really not clear enough

File: pmm-app/tests/verifyInsightDashboards_test.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ Scenario(
     await dashboardPage.expandEachDashboardRow();
     dashboardPage.verifyMetricsExistence(dashboardPage.prometheusExporterStatusDashboard.metrics);
     await dashboardPage.verifyThereAreNoGraphsWithNA(4);
-    await dashboardPage.verifyThereAreNoGraphsWithoutData(14);
+    // increasing to 15 because of failings
```
Comment (puneet0191) [inline pmm-app/tests/verifyInsightDashboards_test.js:31]: The same here, please create one single ticket to add these skips in checks.

**Review by puneet0191 — CHANGES_REQUESTED** [review]

**Comment by IvanaBrecherova** [comment]
https://jira.percona.com/browse/PMM-7502 task for future investigation.

**Comment by yurkovychv** [comment]
@IvanaBrecherova please change the base branch to `release/2.15`

**Review by puneet0191 — APPROVED** [review]

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**